script.js

```diff
@@ -1,5 +1,21 @@
-function addNewItem(foodName, positive) {
-    console.log("Adding new item!", foodName, positive)
+
+function getButton(value)
+{
+    console.log("Pressing button.", value)
+    addNewItem(document.getElementById("foodForm").food.value, value) 
+}
+
+function addNewItem(foodName, value){
+    var today = new Date()
+    var dd = today.getDate()
+    var mm = today.getMonth()
```

Comment:
`getMonth` returns zero-indexed number (means January = 0, December = 11), you probably want to add +1 here.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getMonth

(This is different from getDate that returns days starting from 1.)